services/QuillLMS/vite.config.ts

```diff
@@ -37,48 +44,43 @@ export default defineConfig(({command, mode}) => {
       }
     },
 
-    define: {
-      // global: {}
-      //process: { env: {}}
-
-      // Note: declare node-accessible variable (i.e. CDN_URL) in .env.<mode>, not here
-      // 'process': {
-      //   env: {
-      //     DEFAULT_URL: 1//import.meta.env.DEFAULT_URL
-      //   }
-      // },
-      //'FOOBAR': env.LOCAL_TEST_ENV_VALUE, // not visible from js code,
-      //'process.env.LOCAL_TEST_ENV_VALUE': `"${env.LOCAL_TEST_ENV_VALUE}"` // not visible
-
+    css: {
+      preprocessorOptions: {
+        scss: {
+          additionalData: `@import "./app/assets/stylesheets/variables.scss";`
```

Comment:
This line is super important, because the Vite build was not loading scss partials in the correct order, resulting in 'scss variable not found' errors in the build process.

Reply:
Nice work figuring this out!